src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -38,12 +39,12 @@ def __init__(self, resolution=0.01, feature_dim=3):
         self.feats = None
         self.observations = []
 
-    def add(self, camera_pose: np.ndarray, xyz: np.ndarray, feats: np.ndarray):
+    def add(self, camera_pose: np.ndarray, xyz: np.ndarray, feats: np.ndarray, **info):
```

Comment:
Nit, feel free to ignore: Not sure if it's just me, but for optional info fields I prefer an explicit `info: Dict[str, Any]` instead of using **kwargs, for easier documentation and code tracing.

Reply:
I think this is a fair point, as much as i prefer the syntax for kwargs. Will change this and then merge

Reply:
eh, i actually would rather hold off on this, i think it feels more flexible right now